### PR TITLE
CB-15370 setting opentracing.jdbc.enabled false to prevent log trashing

### PR DESCRIPTION
**Pull request by lnardai**
- need to check it this solves the problem in integration test context

### CODE REVIEW

**Review by lajosrodek — APPROVED** [review]

**Review by biharitomi — APPROVED** [review]

**Comment by lnardai** [comment]
Did not work, integration-test log is still filled with `AppendCommand execution failed! Repeated errors of this command will not be logged.`

**Comment by attilapalfi92** [comment]
Try disabling it here
https://github.com/hortonworks/cloudbreak/blob/master/core/src/main/java/com/sequenceiq/cloudbreak/conf/DatabaseConfig.java#L103

and in all the DatabaseConfig classes for each service

**Comment by lnardai** [comment]
That didnt work. Need to look into it.

**Comment by attilapalfi92** [comment]
Interesting. Try explicitly providing a different JDBC Driver and remove the tracing part from the JDBC URL when JDBC Tracing is disabled.

File: autoscale/src/main/java/com/sequenceiq/periscope/config/DatabaseConfig.java [hunk 1]
```diff
@@ -100,6 +104,9 @@ public DataSource dataSource() throws SQLException {
         if (periscopeNodeConfig.isNodeIdSpecified()) {
             config.addDataSourceProperty("ApplicationName", periscopeNodeConfig.getId());
         }
+        if (tracingConfiguration.isJdbcTracingEnabled()) {
+            config.setDriverClassName("io.opentracing.contrib.jdbc.TracingDriver");
```
Comment (attilapalfi92) [inline autoscale/src/main/java/com/sequenceiq/periscope/config/DatabaseConfig.java:108]: Did it solve the issue?

**Comment by lnardai** [comment]
Not solved the original problem. Need to be investigated further. Closing for now.

**Pull request closed without merging**